### PR TITLE
Fixes for activity report

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for #1055 

Screenshot 2023-09-20 at 4 51 37 PM

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 11 min

File: django_project/frontend/src/containers/MainPage/Data/index.tsx [hunk 1]
```diff
@@ -39,21 +39,21 @@ const DataList = () => {
     const [tableData, setTableData] = useState<any>()
     const [activityTableGrid, setActivityTable] = useState<any>()
     const [userRole, setUserRole] = useState<string>('')
-    const dataset = (userRole ==="Organisation member" || userRole ==="Organisation manager") ?  data.filter(item => !item?.Activity_report)?.flatMap((each) => Object.keys(each)) : data.flatMap((each) => Object.keys(each));
+    const dataset = (userRole ==="Organisation member" || userRole ==="Organisation manager" || userRole === "National data scientist" || userRole === "Regional data scientist") ?  data.filter(item => !item?.Activity_report)?.flatMap((each) => Object.keys(each)) : data.flatMap((each) => Object.keys(each));
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/Data/index.tsx:42]: Please create a function here to check user_role.

**Review by dimasciput — CHANGES_REQUESTED** [review]

File: django_project/frontend/src/containers/MainPage/Data/index.tsx (hunk 1 above)
Comment (ketankartoza) [reply django_project/frontend/src/containers/MainPage/Data/index.tsx:42]: Okay

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 4 min 47 sec

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 18 min

**Pull request merged**